docs/multiwiki.md

```diff
@@ -0,0 +1,60 @@
+Background
+==========
+This application was originally written for English Wikipedia, and for
+convenience the primary IDs of many entities were kept in sync with MediaWiki
+database IDs.
+
+Multiwiki feature
+==================
+In February 2016, we've added support for articles from any of the [Wikimedia
+project wikis](https://meta.wikimedia.org/wiki/Wikimedia_wikis#Core_projects),
+which involves database schema and code changes.  The `Wiki` table will hold
+one record for each wiki referenced from the application, and all onwiki
+entities will link to that row.  The entity primary IDs will be decoupled from
```

Comment:
When you say "all onwiki entities will link to that row", maybe you mean something like, "all Dashboard entities that refer to an entity on a given wiki, or that must specify a wiki for any reason, will link to the corresponding row in the wiki table"? Rrrrg sorry for curmudgeonliness.